cmds/graph.c

```diff
@@ -695,8 +696,8 @@ static void build_graph(struct uftrace_opts *opts, struct uftrace_data *handle,
 		struct uftrace_graph_node *node;
 
 		list_for_each_entry(node, &graph->ug.root.head, list) {
-			graph->ug.root.time += node->time;
-			graph->ug.root.child_time += node->time;
+			graph->ug.root.total_time.sum += node->total_time.sum;
+			graph->ug.root.child_time += node->child_time;
```

Comment:
This is a behavior change.  Since they are top-level nodes, It should use total_time.sum for root's child time.